lib/person_cards.rb

```diff
@@ -23,6 +23,14 @@ def image
     person.image
   end
 
+  def legislature
+    term.legislature
+  end
+
+  def top_identifiers
+    term.top_identifiers
+  end
+
```

Comment:
Good point - thanks. I've force-pushed a version that changes those two methods to be private.